doc/source/user/install.rst

```diff
@@ -1,10 +0,0 @@
-****************
-Installing NumPy
-****************
-
-In most use cases the best way to install NumPy on your system is by using a
-pre-built package for your operating system.  Please see
-https://scipy.org/install.html for links to available options.
-
-For instructions on building for source package, see
-:doc:`building`. This information is useful mainly for advanced users.
```

Comment:
In case users are still linking to this page, should we add a link from here to the Hugo page?

Reply:
@eric-wieser I will do this unless somebody argues otherwise. The reason I've hesitated is that the page might turn up in search and draw attention away from the real installation page. No lasting harm; users will get the Sphinx page and click through to the Hugo page. The issue might not even come up; engines seem to downrank pages that are only a link.